pkg/claims/output.go

```diff
@@ -37,6 +39,24 @@ func (o Output) GetSchema(b cnab.ExtendedBundle) (definition.Schema, bool) {
 	return definition.Schema{}, false
 }
 
+// FormatSensitive transforms an output value to a secret key that can be used
+// to store/retrieve the sensitive output value from a secret store.
+// It returns the transformed output.
+func (o Output) FormatSensitive() Output {
```

Comment:
nit: This name implies that it's formatting the output, but it's actually generating a secretkey, let's update the name to better matches what it's doing (the actual output value isn't actually involved at all).